src/mbgl/gl/context.hpp

```diff
@@ -81,7 +81,10 @@ class Context : private util::noncopyable {
 
     template <RenderbufferType type>
     Renderbuffer<type> createRenderbuffer(const Size size) {
-        static_assert(type == RenderbufferType::RGBA || type == RenderbufferType::DepthStencil,
+        static_assert(type == RenderbufferType::RGBA ||
+                      type == RenderbufferType::DepthStencil ||
+                      type == RenderbufferType::DepthComponent ||
+                      type == RenderbufferType::RGBA4,  // TODO I added RGBA4 because in JS RGBA8 (here 'RGBA') emits a "renderbufferStorage: invalid internalformat" warning (though still renders fine) -- investigate
```

Comment:
Given that we don't seem to need a color renderbuffer (since we're only rendering to a texture), we can remove this and not worry about initialization OpenGL extensions.